app/stylesheets/pages/gradebook2/_gradebook2.scss

```diff
@@ -418,6 +418,33 @@ $cell_height: 33px;
   }
 }
 
+.gradebook-cell-vericite {
+  position: absolute;
+  top: 2px;
+  right: 2px;
+  z-index: 1;
+  width: 20px;
+  height: 16px;
+  &.no-score {
+    background-image: url(/images/vericite_no_score.png);
+  }
+  &.none-score {
+    background-image: url(/images/vericite_none_score.png);
+  }
+  &.acceptable-score {
+    background-image: url(/images/vericite_acceptable_score.png);
+  }
+  &.warning-score {
+    background-image: url(/images/vericite_warning_score.png);
+  }
+  &.problem-score {
+    background-image: url(/images/vericite_problem_score.png);
+  }
+  &.failure-score {
+    background-image: url(/images/vericite_failure_score.png);
+  }
+}
```

Comment:
Lines 428-445: the existing pngs should probably be used.  The only differences appear to be opacity?  If changes are desired a discussion with Instructure's designers should probably be had.